### ISSUE TITLE
Box Links use h3 even if there is no heading set in the box links listing container

### ISSUE THREAD

**Issue by markconroy** [localgovdrupal/localgov_paragraphs#131]
We should have a `h2` for our box links listing heading - which we do if we use one, so that our box links headings are then using `h3`.

However, if we don't set a heading, then we have a set of `h3`s without a parent h2.

## Potential solution? 
Add a boolean field "make heading visible to screenreaders only?" so we can have a h2 that's invisible, but corrects the semantics.

**Comment by markconroy** [comment]
Looks like this is working as expected. Closing.
